pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/PyCoverageTask.groovy

```diff
@@ -34,10 +34,14 @@ import java.util.regex.Pattern
 class PyCoverageTask extends PyTestTask {
 
     @OutputDirectory
-    File coverageOutputDir = project.file("${ project.buildDir }/coverage")
+    File getCoverageOutputDir() {
+        return project.file("${ project.buildDir }/coverage")
+    }
 
     @OutputFile
```

Comment:
Do we need both an `@OutputDirectory` _and_ an `@OutputFile`? The answer relies in whether `PyCoverageTask` has a "monopoly" on the `build/coverage` folder. If so, we can get away with just `@OutputDirectory`.